test/k8s-integration/driver-config.go

```diff
@@ -160,7 +160,6 @@ func generateDriverConfigFile(testParams *testParameters) (string, error) {
 		SnapshotClassFile:         absSnapshotClassFilePath,
 		SnapshotClass:             snapshotClassName,
 		VolumeAttributesClassFile: absVacFilePath,
-		VolumeAttributesClass:     vacName,
```

Comment:
Can we remove the vacName variable now that it's not being used?

Reply:
Yes